### ISSUE TITLE
Clicking on a selected tab removes focus from terminal.

### ISSUE THREAD

**Issue by Blast-City** [elementary/terminal#494]
1- Open two tabs.
2- Click on the selected tab.

**Result:** The text cursor stops flashing and the terminal looses focus. This doesn't happen if on step "2" an user selects other tab instead of the already selected one.

Is this deliberate?

**Comment by jeremypw** [comment]
I am not seeing this on the master version of Terminal.

**Comment by jeremypw** [comment]
Nor on a reference stable installation (other than when the context menu is open of course)

**Comment by Blast-City** [comment]
> Nor on a reference stable installation (other than when the context menu is open of course)

You're right. It only happens when Hide When Single Tab is enabled. I edited the post.

Thanks.

**Comment by jeremypw** [comment]
Seems to be a "feature" of Granite.DynamicNotebook.

**Comment by jeremypw** [comment]
Recompiling Code (which also uses Granite DynamicNotebook) with TabBarBehavior.SINGLE produces the same issue.  Either need to fix Granite or retire the "SINGLE" option in the Terminal settings.
